uint8 point_num
uint8 task_command
float64[] UTMx
float64[] UTMy

